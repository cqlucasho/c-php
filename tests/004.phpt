--TEST--
Check for ant presence
--SKIPIF--
<?php if (!extension_loaded("ant")) print "skip"; ?>
--FILE--
<?php
$database = new AntDatabase('127.0.0.1:3306', 'root', 'root', 'test');
$ret = $database->query('select * from admin')->getRows();
var_dump($ret);
$success = $database->insert('admin',
    array('name' => 'lucasho',
        'nick_name' => 'lucas',
        'password' => '32134',
        'status' => 1,
        'last_ip' => 127001,
        'created' => '2017-11-11 13:11:11'));
echo 'insert:',$success,"\n";

$id = $success->getLastId();
echo 'last id:',$id,"\n";
?>
--EXPECT--
ant extension is available
