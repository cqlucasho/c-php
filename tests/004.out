Fatal error: Call to a member function query() on unknown in /opt/vendor/php/php-5.6.31/ext/ant_compile/tests/004.php on line 3